packages/blocks/src/api/validation.js

```diff
@@ -380,8 +380,27 @@ export function getNextNonWhitespaceToken( tokens ) {
 }
 
 /**
- * Returns true if there is given HTML strings are effectively equivalent, or
- * false otherwise.
+ * Tokenize an HTML string, gracefully handling any errors thrown during
+ * underlying tokenization.
+ *
+ * @param {string} html HTML string to tokenize.
+ *
+ * @return {Object[]|boolean} Array of valid tokenized HTML elements, or false
```

Comment:
The documentation should describe in what cases the consumer would expect the function to return `false`. Currently it's not clear that this is tied to capturing errors.

Reply:
Aside: Why was `false` chosen, vs. another more semantically meaningful value for empty like `null` or `undefined` ?

Reply:
> Why was false chosen, vs. another more semantically meaningful value for empty like null or undefined

Force of habit from PHPland. Swapped to `null`